lizard_auth_client/client.py

```diff
@@ -238,7 +238,9 @@ def sso_authenticate_django_v2(username, password):
             'domain': None,
```

Comment:
Eh nee, het hoeft *niet* ingesteld te worden. Dat domain is nodig als er na afloop door de SSO weer terug wordt geredirect naar de site. Dàn moet de juiste domeinnaam gebruikt worden. Nu hoeft  het niet.

Ik weet weer waarom ik het op een default heb staan: in de server code is/was het een verplicht veld. Misschien ga ik toch wat meer naar eigen inzicht opschonen aan die kant...